ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestionProperties.java

```diff
@@ -91,6 +123,9 @@ public IngestionProperties(String databaseName, String tableName)
         QueueAndTable
     }
 
+    /**
+     * Validate the minimum non-empty values needed for data ingestion.
+     */
     public void validate(){
```

Comment:
This shouldn't be used by users, right? It can be package-private...

Reply:
Changed